ArcSlider/ArcSlider.js

```diff
@@ -188,6 +204,7 @@ const ArcSliderBase = kind({
 		foregroundColor: '#0000ff',
```

Comment:
> > Did I something wrong?
Why the silicon default color became blue?
@adrian-cocoara-lgp @paul-beldean-lgp

> It seems the default blue color comes from the default ArcSlider prop "foregroundColor: '#0000ff'" which was last changed half a year ago. Also the colors for the slider don't come from from the skin styles, they come from the props 'backgroundColor' and 'foregroundColor', the only style set by the theme is the knob color on focus and that comes from the 'defaultColors' in ThemeDecorator\ThemeDecorator.js/

I meant, the default color is  white in GUI guide, so how about change this part?
some components a kind of ArcPicker and ArcSlider don't have GUI guidem, except for silicon. So you can modify them to be the same as silicon except for the accent and highlight color you mention.